mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx

```diff
@@ -0,0 +1,41 @@
+import ReactDOM from 'react-dom';
+import { HashRouter } from 'react-router-dom';
+import { TopBarWithBeamer } from '../../../../common/top_bar/top_bar';
+import { Notices } from '../../../listing/components/notices';
+import { Header } from './components/header';
+import { Overview } from './components/overview';
+import { Tabs } from './components/tabs';
+import { createStore } from './store';
+import { boot } from './helpers/boot';
+import { registerApiErrorHandler } from '../../../listing/api-error-handler';
+
+function Analytics(): JSX.Element {
+  return (
+    <div className="mailpoet-automation-analytics">
+      <Header />
+      <Overview />
+    </div>
+  );
+}
+
+function App(): JSX.Element {
+  return (
+    <HashRouter>
+      <TopBarWithBeamer />
+      <Notices />
+      <Analytics />
+      <Tabs />
```

Comment:
It feels somewhat strange that these `<Tabs>` are not within the `Analytics` component. They also seem to be analytics-specific tabs, so maybe the generic `Tabs` naming is also not ideal.

Reply:
Thanks! Moved.